tests/consistent-package-json.test.ts

```diff
@@ -75,6 +75,11 @@ packages.forEach(
             // We want to make sure the first set of items are always in a fixed
             // order. Most importantly, the `!*.tsbuildinfo` exclusion must be
             // after `build/*`.
+            /* eslint-disable jest/no-if */
+            if (packageJSON?.bin) {
```

Comment:
hacky but only some of the packages need to export `bin/*` with the files entry in `package.json` we could also break this into to sets of looping tests.